package.json

```diff
@@ -20,6 +20,7 @@
         "@khanacademy/wonder-blocks-button": "catalog:",
         "@khanacademy/wonder-blocks-core": "catalog:",
         "@khanacademy/wonder-blocks-icon": "catalog:",
+        "@khanacademy/wonder-blocks-labeled-field": "catalog:",
```

Comment:
We also don't need this as a dependency here. I need to clean these up, but we should only have tooling and storybook-required dependencies listed here.  Each package defines the deps it needs in its own `package.json`. Sorry for the confusion.